dbt/seeds/etl_full_row_counts.csv

```diff
@@ -3461,6 +3464,7 @@ out_ferc714__respondents_with_fips,2020,9032
 out_ferc714__respondents_with_fips,2021,8976
 out_ferc714__respondents_with_fips,2022,8987
 out_ferc714__respondents_with_fips,2023,8998
+out_ferc714__respondents_with_fips,2024,218
```

Comment:
This is expected to be small, since without the EIA-861 data for 2024, there are no 2024 service territories.